metaci/repository/models.py

```diff
@@ -34,7 +34,7 @@ class Repository(models.Model):
     github_id = models.IntegerField(null=True, blank=True)
     url = models.URLField(max_length=255)
     release_tag_regex = models.CharField(max_length=255, blank=True, null=True)
-    default_implementation_steps = models.JSONField(null=True, default=[])
+    default_implementation_steps = models.JSONField(null=True, default=list)
```

Comment:
These last 2 files duplicate the changes in #2487; were you intending to submit them there or here?